packages/flutter_notification_center_firebase/lib/src/services/firebase_notification_service.dart

```diff
@@ -62,8 +72,17 @@ class FirebaseNotificationService
   @override
   Future<List<NotificationModel>> getActiveNotifications() async {
     try {
+      var userId = FirebaseAuth.instance.currentUser?.uid;
```

Comment:
Can you add an option to give a specific firebase app or firebaseAuth to the service